routes/data/strings.js

```diff
@@ -0,0 +1,48 @@
+const commonStrings = {
+    header:{
+        "Home_Menu" : "Home",
+        "Product_Menu" : "Product",
+        "Login_Menu" : "SignIn",
+        "Register_Menu" : "Register",
+        "Cart_Item" : "Item"
+    },
+
+    footer:{
+        "Copyright" : "Copyright © 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd"
```

Comment:
try making the year dynamic
`Copyright © 2011- ${currentYear} Sabka Bazaar Grocery Supplies Pvt Ltd`